_data/developer.yml

```diff
@@ -315,6 +315,13 @@ websites:
       software: Yes
       doc: http://support.beanbaginc.com/support/solutions/articles/3000006998-enabling-two-factor-authentication
 
+    - name: Read the Docs
+      url: https://readthedocs.org
+      img: readthedocs.png
+      tfa: No
+      twitter: readthedocs
```

Comment:
Doesn't seem like they're monitoring their twitter account. I suggest we only use the email address.